integration-box/optimizely/readme.md

```diff
@@ -0,0 +1,108 @@
+# Integrating TD Profiles API with Optimizely
+
+You can integrate Treasure Data's Profiles API with Optimizely Web Experimentation SDK to create a personalized experience for your customers. Optimizely Web Experimentation is an experimentation platform that allows you to test real visitor experiences on your site using a variety of options including A/B testing, multivariate testing, and personalized campaigns. Optimizely empowers you to make better decisions by improving customer experiences through experimentation.
+
+## Prerequisites 
+
+- Basic knowledge of JavaScript and HTML
+- Basic knowledge of Treasure Data
+- Basic knowledge of Treasure Data JavaScript SDK
+- Basic knowledge of Treasure Data Profiles API
+- Basic knowledge of Optimizely Web Experimentation
+
+Use of Treasure Data JavaScript SDK might require the direction of the Treasure Data Support or Services team.
```

Comment:
We need to put ... This integration is a custom integration by our solution architect team. If need any questions, please reach out to your customer success maanger.